local-tests/tests/testUseEoaSessionSigsToRequestSingleResponse.ts

```diff
@@ -0,0 +1,55 @@
+import { getEoaSessionSigs } from 'local-tests/setup/session-sigs/get-eoa-session-sigs';
+import { TinnyEnvironment } from 'local-tests/setup/tinny-environment';
+
+/**
+ * Test Commands:
+ * ✅ NETWORK=datil-dev yarn test:local --filter=testUseEoaSessionSigsToRequestSingleResponse
+ * ✅ NETWORK=datil-test yarn test:local --filter=testUseEoaSessionSigsToRequestSingleResponse
+ * ✅ NETWORK=datil yarn test:local --filter=testUseEoaSessionSigsToRequestSingleResponse
+ */
+export const testUseEoaSessionSigsToRequestSingleResponse = async (
+  devEnv: TinnyEnvironment
+) => {
+  const alice = await devEnv.createRandomPerson();
+
+  const eoaSessionSigs = await getEoaSessionSigs(devEnv, alice);
+
+  const res = await devEnv.litNodeClient.executeJs({
+    sessionSigs: eoaSessionSigs,
+    code: `(async () => {
+      console.log('hello world')
+    })();`,
+    numResponsesRequired: 1,
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
```

Comment:
to ensure we don't leave a key hanging if this fails for whatever reason, the release needs to go in a finally{} block, e.g.
```suggestion
  try {
      const eoaSessionSigs = await getEoaSessionSigs(devEnv, alice);
    
      const res = await devEnv.litNodeClient.executeJs({
        sessionSigs: eoaSessionSigs,
        code: `(async () => {
          console.log('hello world')
        })();`,
        numResponsesRequired: 1,
      });
    } finally {
      devEnv.releasePrivateKeyFromUser(alice);
    }
```